readme-gen.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPTPATH=$(dirname "$0")
+README_GENERATOR_VERSION="2.5.1"
+README_GENERATOR_EXE="readme-generator"
+
+# platform agnostic npm install, also adds into the path
+npm install -g "@bitnami/readme-generator-for-helm@${README_GENERATOR_VERSION}"
+
+if [[ -z '$(type -d "${README_GENERATOR_EXE}")' ]]; then
+  echo "readme-generator not installed successfully!!"
+fi
+
+# generate docs and show the diff
+readme-generator --values="${SCRIPTPATH}/charts/spire/values.yaml" --readme="${SCRIPTPATH}/charts/spire/README.md"
```

Comment:
Maybe we should loop all folders containing the Chart.yaml to generate docs for all charts (subcharts right now, but later on all the top level charts).

Reply:
Done and tested. This relies on the `find`, `dirname` and `xargs` utilities.